style.css

```diff
@@ -0,0 +1,581 @@
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: lato, sans-serif;
+}
+
+section {
+  padding: 1rem;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  text-transform: capitalize;
+}
+
+.upper {
+  display: none;
+}
+
+header {
+  background-image: url(images/bgtest.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 1rem;
+  padding-bottom: 4rem;
+}
+
+.logo {
+  display: none;
+}
+
+.icon i {
+  font-size: 30px;
+  color: #272a31;
+}
+
+nav ul {
+  position: absolute;
+  height: 100vh;
+  width: 60%;
+  background-color: rgb(235, 230, 230);
+  top: 0;
+  right: -100%;
+  padding: 2rem 1rem;
+  display: none;
+  transition: 0.5s linear;
+  z-index: 99;
+}
+
+.icon .fa-xmark {
+  display: none;
+}
+
+nav li {
+  margin-bottom: 2rem;
+  transition: 0.4s linear;
+}
+
+nav a {
+  font-size: 23px;
+  color: rgb(58, 58, 58);
+  transition: 0.3s linear;
+}
+
+.btn a {
+  display: none;
+}
+
+nav a:hover {
+  color: #ec5242;
+}
+
+.links {
+  text-transform: capitalize;
+}
+
+footer h3 {
+  border: 2px solid #ec5242;
+  text-align: center;
+  padding: 1px 1rem;
+  font-size: 15px;
+  margin-right: 1rem;
+  color: #272a31;
+}
+
+footer p {
+  font-size: 14px;
+  margin-left: 1rem;
+  color: #272a31;
+}
+
+section h2 {
+  text-align: center;
+  margin: 2rem 0;
+  position: relative;
+}
+
+section h2::after {
+  content: '';
+  width: 35px;
+  height: 2px;
+  border-radius: 12px;
+  position: absolute;
+  right: calc(50% - 8px);
+  bottom: -8px;
+  background-color: #ec5242;
+}
+
+.intro {
+  padding: 2.5rem 0;
+  text-transform: capitalize;
+}
+
+.intro h3 {
+  font-size: 23px;
+  color: #ec5242;
+  font-weight: 400;
+}
+
+.intro h2 {
+  font-size: 28px;
+  margin-top: 15px;
+  font-weight: 800;
+}
+
+.intro h1 {
+  color: rgb(236 12 12 / 86%);
+  font-size: 32px;
+  line-height: 1.1;
+  margin: 0.5rem 0;
+  text-transform: uppercase;
+}
+
+.pros p {
+  width: 65%;
+}
+
+.intro p:last-child,
+.intro h2 {
+  color: #272a31;
+}
+
+.bg {
+  background-color: rgb(245 243 243 / 93%);
+  border: 2px solid white;
+  padding: 10px;
+  margin: 1rem 0;
+}
+
+.show {
+  right: 0;
+  display: block;
+}
+
+.how .fa-bars {
+  display: none;
+}
+
+.how .fa-xmark {
+  display: block;
+}
+
+#larger {
+  display: none;
+}
+
+.program {
+  padding: 2rem 1rem;
+  background-image: url(images/program-mobile.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.program h2 {
+  text-align: center;
+  text-transform: capitalize;
+}
+
+.container {
+  display: grid;
+  justify-content: center;
+}
+
+.pros {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 5px 0;
+  padding: 5px;
+  background-color: rgba(255, 255, 255, 0.288);
+  transition: 0.3s linear;
+}
```

Comment:
- [ ] The text for each card is not aligned properly with texts from other cards in the _**Program Section**_. This makes the cards not identical to that of the Behance template provided. Kindly realign it. 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/210666739-eb791e66-6da3-433e-a042-5c61290e4aeb.png)

### Template Design

![image](https://user-images.githubusercontent.com/77106962/210666844-2fe85ef5-b80f-457f-93cc-6629b2056c37.png)

#### Hint: I would advise that you remove the `justify-content` declaration and use the `gap` declaration. 👍 😄